ios-target/View/Home/TargetFormViewController.swift

```diff
@@ -62,6 +69,23 @@ extension TargetFormViewController: TargetFormViewModelDelegate, TargetActionsDe
   }
   
   func formDidChange() {
-    createTargetButton.enableButton(enable: viewModel.hasValidData)
+    createTargetButton.enableButton(viewModel.hasValidData)
+  }
+}
+
+extension TargetFormViewController: TopicViewModelDelegate {
+  func topicSelected(topic: Topic) {
+    selectTopicButton.setTitle(topic.label.uppercased(), for: .normal)
+    selectTopicButton.setImage(topic.image?.withRenderingMode(.alwaysTemplate),
+                               for: .normal)
+    selectTopicButton.tintColor = .black
+    selectTopicButton.titleEdgeInsets = UIEdgeInsets(top: 0,
```

Comment:
be consistent in the way you break params. In this case I'd go in a single line, like line 86